typings/asyncQeueu.ts

```diff
@@ -0,0 +1 @@
+declare module 'async/queue';
```

Comment:
This filename is spelled incorrectly, it should be asyncQueue.ts

Btw, why is this necessary? Could we remove this file and in bin.ts do:

```typescript
import {queue} from 'async';
```